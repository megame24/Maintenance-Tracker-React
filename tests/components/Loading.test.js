import React from 'react';
import { shallow } from "enzyme";
import Loading from '../../src/components/Loading';

describe('Testing Loading component', () => {
  it('should render as expected', () => {
    const tree = shallow(<Loading isLoading={false} />);
    expect(tree).toMatchSnapshot();
  });
});
