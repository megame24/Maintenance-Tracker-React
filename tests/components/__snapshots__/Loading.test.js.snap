// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Loading component should render as expected 1`] = `
<Dimmer
  active={false}
  page={true}
>
  <Loader
    size="medium"
  />
</Dimmer>
`;
